### PR TITLE
Front page

### PR DESCRIPTION
**Pull request by redacuve**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,100 @@
...
+            <img src="resources/iphone5s.jpg" alt="iPhone 5s">
+        </div>
+        <div>
+            <img src="resources/ipgone5c.jpg" alt="iPhone 5c">
+        </div>
+        <div>
+            <img src="resources/mac-birthday.jpg" alt="30 years of Mac">
+        </div>
+        <div>
+            <img src="resources/supplier-responsability.jpg" alt="We got supplier responsability">
```
Comment (suretrust) [inline index.html]: Each of these images should be wrapped with an `<a></a>` tag. They are links on the original web page.

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @redacuve,
### Great job achieving a near-perfect gradient. :raised_hands:  Just a few suggestions to wrap it up:

- [ ] See my comment on your code below.

- [ ] There is a border in between each nav button, let's try to achieve this and style the search input as shown on the Apple page. `[Optional]` Each button is missing styling on hover.

- [ ] Let's put the width of the upper and lower section in alignment. Please refer to the image below:

align

**Kindly submit for another code review once these issues have been fixed.** _Happy coding! :tada:_

File: index.html [hunk 2]
```diff
@@ -1,13 +1,100 @@
...
+    </div>
+
+
+    <aside class="grid-aside">
+        <div>
+            <a href="http://www.apple.com/iphone-5s/"><img src="resources/iphone5s.jpg" alt="iPhone 5s"></a>
+        </div>
+        <div>
+            <a href="http://www.apple.com/iphone-5c/"><img src="resources/ipgone5c.jpg" alt="iPhone 5c"></a>
+        </div>
```
Comment (shloch) [inline index.html:46]: good job with the corrections

**Review by shloch — COMMENTED** [review]
### Greetings  @redacuve 👏👏👏

Your coding is simple and very efficient

Instead of adding "PROJECT LINK" into your README file (which is just a link into the repo, which we're already in ), it's better to add link to your LIVE DEMO. I couldn't see that.

Since we're aiming to be as professional as possible, please add the live demo link into your README. I generated one for you using [https://raw.githack.com](https://raw.githack.com) --> `https://raw.githack.com/redacuve/FakeApple/Front-Page/index.html`

I count on you to fix this before merging this pull request 😊😊

### THIS PROJECT IS CONSIDERED VALID 👍👍👍

**Pull request merged**